docs/py_docs.rst

```diff
@@ -677,21 +747,23 @@ References cited in the :ref:`'Notes' <py-docstring-notes>` section may be liste
       and neural-network techniques," Computers & Geosciences, vol. 22,
       pp. 585-588, 1996.
 
-Note that Web pages should be referenced with regular inline links.
+Web pages should be referenced with regular inline links.
```

Comment:
In the line above the example, "as in the list as follows" should perhaps be simply "as follows".